src/main/java/frc/robot/subsystems/drivetrain/BaseSwerveSubsystem.java

```diff
@@ -224,26 +224,38 @@ public void resetPosition() {
 
     /**
      * Zeros *only the angle* of the robot's field-relative control system.
-     * This this has *no effect* on odometry.
+     * This also resets localization to match the rotated field.
      */
     public void resetFieldAngle() {
```

Comment:
the more i think abt it, the more it seems to me that this method is really just resetRotation, a piece of resetPosition. I think we should reset both field-oriented-drive and poseEstimator's heading in both cases, and the difference is in whether we set the translation? idk this is hurting my brain a lil, what do u/ya'll think